docs/introduction/hello-world-VS.md

```diff
@@ -0,0 +1,225 @@
+# Running hello_world example using HELICS windows installer + Visual Studio 2019
+
+1. Make sure that you have downloaded HELICS through the installer.
+
+  - To get to the installer you can access it through either of the following links:
+    * https://github.com/GMLC-TDC/HELICS/releases
+    * https://docs.helics.org/en/latest/installation/index.html -  where you will see this:
```

Comment:
We're re-working the installation guide right now to make it clearer to users. This screen shot might need to be updated once that draft is in place in a few weeks. We can worry about that then, though.

Reply:
And now that I look at preview of the rendered Markdown, I can see that we almost certainly won't need to update that image. Yeah!